src/V3Width.cpp

```diff
@@ -3090,7 +3090,10 @@ class WidthVisitor final : public VNVisitor {
             newp = new AstCMethodHard(nodep->fileline(), nodep->fromp()->unlinkFrBack(),
                                       nodep->name());
             newp->dtypeFrom(adtypep->subDTypep());
-            if (!nodep->firstAbovep()) newp->makeStatement();
+            // because queue methods pop_front() or pop_back() can be void cast,
+            // they use a special check if they need the c++ ";" added (done by
+            // newp->makeStatement())
```

Comment:
```suggestion
            // Because queue methods pop_front() or pop_back() can be void cast,
            // they use makeStatement to check if they need the c++ ";" added
```